### ISSUE TITLE
UnusedUses not compatible with nested ::class references used in Doctrine annotations

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#281]
```php
<?php

namespace Test;

use Foo\Bar\A;
use Foo\Bar\B;
use Foo\Bar\C;
use Foo\Bar\D;

/**
 * @CustomAnnotation(A::class)
 * @CustomAnnotation(prop=B::class)
 * @CustomAnnotation(@AnotherCustomAnnotation(C::class))
 * @CustomAnnotation(prop=@AnotherCustomAnnotation(D::class))
 */
class Foo {}
```

First two are ok, but the other two are not detected:
```
 7 | ERROR | [x] Type Foo\Bar\C is not used in this file. (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
 8 | ERROR | [x] Type Foo\Bar\D is not used in this file. (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
```

**Comment by kukulich** [comment]
Thank for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/09b4ce5c86df9e071bc1e86111a7455c8a4650fa
